### PR TITLE
Research digiboard

### PR DESCRIPTION
**Pull request by SirSmith148**
## About the PR

Added New Mystagogue item, the Research Digi-Board, this work similarly to the Logistics Digi-Board, allowing remote use of the research console. (Link to Pull request of original Digi-board: https://github.com/DeltaV-Station/Delta-v/commit/abaa2feba5b17bde4014c8b7d2525d9f3254671c#diff-6e493dfa3320562d461bc23a070f671cfd6e2f55574c8de3a5c9ea6ed6799bffR557)

## Why / Balance

The Logistics Officer and the Mystagogue have the unique gameplay loop of needing to take requests from the crew for certain items/research, and then using a console to order or approve. Giving the Mysta the ability for it will allow them to fulfill these requests when otherwise busy handling anoms or deep inside the research labs of epi, and when understaffed. 

## Technical details

-Added code for opening the research menu with the Digi-Board
-Added new Digi-Board to antag steal objective lists
-Added new Digi-Board to Mysta Locker list

## Media

Just a recolor of the original Digi-Board Sprite
https://imgur.com/a/1kCcJDx

## Requirements

- [X] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

N/A

**Changelog**

:cl:
- add: Research Digi-Board
- tweak: Added Research Digi-Board to Steal Objectives

### CODE REVIEW

**Review by deltanedas — CHANGES_REQUESTED** [review]
all of these need to go in deltav folders

**Comment by SirSmith148** [comment]
> all of these need to go in deltav folders

**Comment by deltanedas** [comment]
you dont have to close it or anything just push commits to the same branch

**Comment by SirSmith148** [comment]
> you dont have to close it or anything just push commits to the same branch

It was a mis click sorry haha, I only really edited existing files, do I need to migrate the whole files I edited to the DeltaV folder?

**Comment by deltanedas** [comment]
yes

**Comment by SirSmith148** [comment]
> yes

Updated was able to move everything to Delta V except the ObjectiveGroups as I saw all the other DeltaV ones were in the original file

File: Resources/Prototypes/DeltaV/Entities/Objects/Misc/paper.yml
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Objects/Misc/paper.yml]: dont need /Textures/ prefix

File: Resources/Prototypes/DeltaV/Entities/Objects/Misc/paper.yml
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Objects/Misc/paper.yml]: remove one of the extra lines

File: Resources/Prototypes/Entities/Objects/Misc/paper.yml
Comment (deltanedas) [inline Resources/Prototypes/Entities/Objects/Misc/paper.yml]: untouch

File: Resources/Prototypes/Objectives/objectiveGroups.yml
Comment (deltanedas) [inline Resources/Prototypes/Objectives/objectiveGroups.yml]: add a space after the #

File: Resources/Textures/DeltaV/Objects/Misc/rd_clipboard.rsi/meta.json
Comment (deltanedas) [inline Resources/Textures/DeltaV/Objects/Misc/rd_clipboard.rsi/meta.json:1]: 4 space indent not 2

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Textures/DeltaV/Objects/Misc/rd_clipboard.rsi/meta.json
Comment (deltanedas) [inline Resources/Textures/DeltaV/Objects/Misc/rd_clipboard.rsi/meta.json:1]: replace tabs with spaces

**Review by deltanedas — CHANGES_REQUESTED** [review]
other than that, just waiting on direction approval

**Comment by Lyndomen** [comment]
Hello! Direction review, 24 hours.

**Comment by SirSmith148** [comment]
> Hello! Direction review, 24 hours.

Thank you!

**Review by deltanedas — DISMISSED** [review]

**Review by Lyndomen — CHANGES_REQUESTED** [review]
Requesting an update to move this down to about 7 or 8 blunt, not 10. Once that is all set, approved!

**Comment by SirSmith148** [comment]
> Requesting an update to move this down to about 7 or 8 blunt, not 10. Once that is all set, approved!

Its the same amount of blunt as the logistics version not that it really matters to me lol

**Review by Lyndomen — APPROVED** [review]

**Pull request merged**
